static/js/apps/nl_interface/app.tsx

```diff
@@ -58,12 +64,20 @@ export function App(): JSX.Element {
     setContextList(newList);
   }
 
+  function getContextHistory(i: number) {
```

Comment:
nit: we usually want to specify the type that a function is returning

so `function getContextHistory(i: number): <return type> {}`